### PR TITLE
bloggy post

### PR DESCRIPTION
**Pull request by saschaben**
bloggy post on digital literacy

### CODE REVIEW

**Comment by saschaben** [comment]
@jrjohnson please take a gander at this draft post, and let me know what your feedback is, and then pass on to @thecoolestguy and/or @stopfstedt for further feedback. thx.

File: _posts/2016-02-09-literacy.markdown [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title:  "On Students, Ilios, and Digital Literacy"
+date:   2016-02-09 15:00:00
+categories: news
+---
+One of the most challenging things in using a tool like Ilios actually has nothing to do with the tool at all, really. It is a problem with communication and digital literacy, not just among our faculty and educational staff, but (and most significantly) among our students as well.
+
+Let's face it: Ilios as a tool and a platform is pretty wickedly complex. Lots of moving parts, thousand (if not millions) of lines of code, dozens of dependencies....pretty much like any other contemporary application being used as a solution. Even so, most of the complications, user issues, and challenges we have seen coming to the core Ilios team aren't really about that.
```
Comment (thecoolestguy) [inline _posts/2016-02-09-literacy.markdown]: 'thousand' should be 'thousands'

File: _posts/2016-02-09-literacy.markdown [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+This is disheartening. But what can we do about it?
+
+__Step #1: Better Communication__
+
+Even before we starting handing tools to students to help them in their study, we need to be sure that we are telling them clearly not just how to use the tools, but what those tools even are, and how they fit into the landscape of technology and services they need to add to their experience to be successful. That information should start at day one, and must be both consistent in content and over time. The technology ecosystem we work in is complex; with the rapid proliferation of new tools and processes, changes in social media use and asynchronous learning models, it is only getting more complicated and dense. It is incumbent on us, as educational technologists, to effectively communicate this complexity to our communities. This is easy to say: it is hard to do. Nevertheless, without effective communications about the stuff that makes our stuff work -- the underpinnings of our systems, rather than the tools themselves -- any challenges faced by our users will wind up being both misdirected and ultimately dissatisfied with the solutions they are offered.
```
Comment (jrjohnson) [inline _posts/2016-02-09-literacy.markdown]: `both consistent in content and over time` is a nice phrase, but I had to read it a couple of times to understand it.

Comment (jrjohnson) [inline _posts/2016-02-09-literacy.markdown]: `with the rapid proliferation of new tools and processes, changes in social media use and asynchronous learning models,` is probably gramatically correct (I wouldn't know honestly) but the interpolation of a list an a clause makes it hard to understand.

**Comment by jrjohnson** [comment]
@saschaben Love the message here - comments are minor and specific.

**Comment by saschaben** [comment]
@jrjohnson thanks for the feedback!

@thecoolestguy your input next: if you have anything to add, please do so!

File: _posts/2016-02-09-literacy.markdown (hunk 2 above)
Comment (saschaben) [inline _posts/2016-02-09-literacy.markdown]: good point. I'll rework this.

**Comment by dartajax** [comment]
+Let's face it: Ilios as a tool and a platform is pretty wickedly complex. There are lots of moving parts, thousands (if not millions) of lines of code, dozens of dependencies ... pretty much like any other contemporary application being used as a solution. Even so, most of the complications, user issues, and challenges we have seen coming to the core Ilios team aren't really about that.  

(what does that refer to?) I put "There are" at the beginning of that sentence as a suggestion and put three ...'s.

File: _posts/2016-02-09-literacy.markdown [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title:  "On Students, Ilios, and Digital Literacy"
+date:   2016-02-09 15:00:00
+categories: news
+---
+One of the most challenging things in using a tool like Ilios actually has nothing to do with the tool at all, really. It is a problem with communication and digital literacy, not just among our faculty and educational staff, but (and most significantly) among our students as well.
```
Comment (stopfstedt) [inline _posts/2016-02-09-literacy.markdown]: drop actually, replace really with actually.

File: _posts/2016-02-09-literacy.markdown (hunk 2 above)
Comment (stopfstedt) [inline _posts/2016-02-09-literacy.markdown]: suggesting replacing "about the stuff that makes our stuff work" with "about the things that makes our stuff work".

File: _posts/2016-02-09-literacy.markdown [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+Is this easy? Of course not. But the systems we are building and integrating are really, *really* complicated. The multiplicity of applications, data transport systems, proliferating points of failure, not to mention the basic behavioral and usability patterns that need to be absorbed in order to be effective in using systems like Ilios. As much as we all seek the holy grail of usability for educational systems, that will give students, faculty and staff utterly simple, intuitive, and foolproof ways of interacting with data and systems, so far --- despite the truly remarkable innovations we have seen in the last decade in digital tools development --- so far that grail not only remains elusive, but still appears potentially unachievable without radical simplifications in our pedagogies. And rest assured, that isn't a trend any of us are seeing.
+
+All of this certainly applies to my core team here, developing and supporting Ilios. But it is my opinion an suspicion that it is more broadly applicable to many, if not most of our technology initiatives which touch the lives and activities of our learners and those of us who facilitate and support their progress.
```
Comment (stopfstedt) [inline _posts/2016-02-09-literacy.markdown]: typo: "opinion an suspicion". should be "opinion and suspicion"

**Pull request merged**